components/usage/pkg/apiv1/billing.go

```diff
@@ -92,6 +92,19 @@ func (s *BillingService) GetStripeCustomer(ctx context.Context, req *v1.GetStrip
 			}
 
 			logger.WithError(err).Error("Failed to lookup stripe customer from DB")
+
+			return nil, status.Errorf(codes.NotFound, "Failed to lookup stripe customer from DB: %s", err.Error())
```

Comment:
This line looks like it snuck in here, intentional?

Reply:
yes, I copied the same content in line 140, so an error is returned, not an empty `v1.StripeCustomer`.